### DISCUSSION TITLE
Support smsmode API

### DISCUSSION THREAD

**Discussion by williamdes** [upptime/upptime#242] [Ideas] [answered]
Hi !

Today I moved to your software: https://blog.williamdes.eu/OpenSource/stories/moving-my-uptime-service-to-upptime/

I was using smsmode API, can I implement it here ?

Implementation: https://github.com/wdes/stats/blob/main/src/lib/plugins/sms/SmsModeApi.ts
Website: https://www.smsmode.com/

**Comment by AnandChowdhary** [comment, marked as answer]
Thanks for writing the article, I enjoyed it! I'll add support for this API soon!
